File: app/src/main/java/com/emiliorg/myrestaurant/model/Ingredientes.java
package com.emiliorg.myrestaurant.model;

public class Ingredientes {
    private boolean vegano;
    private boolean gluten;
    private boolean lactosa;
    private boolean picante;

    public Ingredientes() {
    }

    public Ingredientes(boolean vegano, boolean gluten, boolean lactosa, boolean picante) {
        this.vegano = vegano;
        this.gluten = gluten;
        this.lactosa = lactosa;
        this.picante = picante;
    }

    public boolean isVegano() {
        return vegano;
    }

    public void setVegano(boolean vegano) {
        this.vegano = vegano;
    }

    public boolean isGluten() {
        return gluten;
    }

    public void setGluten(boolean gluten) {
        this.gluten = gluten;
    }

    public boolean isLactosa() {
        return lactosa;
    }

    public void setLactosa(boolean lactosa) {
        this.lactosa = lactosa;
    }

    public boolean isPicante() {
        return picante;
    }

    public void setPicante(boolean picante) {
        this.picante = picante;
    }

    @Override
    public String toString() {
        return "Ingredientes{" +
                "vegano=" + vegano +
                ", gluten=" + gluten +
                ", lactosa=" + lactosa +
                ", picante=" + picante +
                '}';
    }
}

File: app/src/main/java/com/emiliorg/myrestaurant/model/Ingredientes.kt
package com.emiliorg.myrestaurant.model

class Ingredientes {
    private var vegano = false
    private var gluten = false
    private var lactosa = false
    private var picante = false

    constructor() {}
    constructor(vegano: Boolean, gluten: Boolean, lactosa: Boolean, picante: Boolean) {
        this.vegano = vegano
        this.gluten = gluten
        this.lactosa = lactosa
        this.picante = picante
    }

    fun isVegano(): Boolean {
        return vegano
    }

    fun setVegano(vegano: Boolean) {
        this.vegano = vegano
    }

    fun isGluten(): Boolean {
        return gluten
    }

    fun setGluten(gluten: Boolean) {
        this.gluten = gluten
    }

    fun isLactosa(): Boolean {
        return lactosa
    }

    fun setLactosa(lactosa: Boolean) {
        this.lactosa = lactosa
    }

    fun isPicante(): Boolean {
        return picante
    }

    fun setPicante(picante: Boolean) {
        this.picante = picante
    }

    override fun toString(): String {
        return "Ingredientes{" +
                "vegano=" + vegano +
                ", gluten=" + gluten +
                ", lactosa=" + lactosa +
                ", picante=" + picante +
                '}'
    }
}